packages/client/src/utils/writeClientSource.ts

```diff
@@ -91,10 +91,10 @@ export function printSourceFile(source: ts.SourceFile): string {
 }
 
 async function exists(pathString: string) {
-  try {
-    await fs.access(pathString)
-    return true
-  } catch {
-    return false
-  }
+  // try {
+  //   await fs.access(pathString)
+  //   return true
+  // } catch {
```

Comment:
Should we remove this?

Reply:
I think so, I'll remove the whole "dont write if we already have it", because it's too much in flux right now to check only on the file to exist. We should rather make an md5 hash or somehting.